app/server/consent/html.ts

```diff
@@ -82,8 +82,6 @@ const html: (
                 e.data.fonts.forEach(useFont);
               }
             });
-
-            document.body.appendChild(iframe);
```

Comment:
This line of font loading logic wasn't required (it was appending an iframe to the document body that already existed in the body), I noticed this error because I saw 2 network requests for the `font-loading` endpoint in the Nework panel.